src/utils/dateUtils.js

```diff
@@ -94,40 +76,58 @@ const dateUtils = {
         return moment.utc(data, 'YYYY-MM-DDTHH:mm:ss').format('L');
     },
 
-    formatTimeOnly(data) {
+    formatDateWithSpecifiedTz(data, timeZoneId) {
         if (_.isNil(data)) {
             return null;
         }
 
         if (_.isNumber(data)) {
-            return moment.unix(data).utc().format('L LT z');
+            return moment.unix(data).utc().tz(timeZoneId).format('MM/DD/YY'); // TODO/FIXME: Consider using L (although that has a 4 digit year, i.e. YYYY)
         }
 
-        return moment(data, 'HH:mm').format('hh:mm a');
+        return moment(data).tz(timeZoneId).format('MM/DD/YY'); // TODO/FIXME: Consider using L (although that has a 4 digit year, i.e. YYYY)
     },
 
-    formatShort(data) {
+    formatDateAndTime(data) {
         if (_.isNil(data)) {
             return null;
         }
 
         if (_.isNumber(data)) {
-            return moment.unix(data).utc().format('MM/DD/YY'); // TODO/FIXME: Consider using L (although that has a 4 digit year, i.e. YYYY)
+            return moment.unix(data).utc().format('L LT');
         }
 
-        return moment(data).format('MM/DD/YY'); // TODO/FIXME: Consider using L (although that has a 4 digit year, i.e. YYYY)
+        return moment.utc(data, 'YYYY-MM-DDTHH:mm:ss').format('L LT');
     },
 
-    formatShortWithSpecifiedTz(data, timeZoneId) {
+    formatDateAndTimeWithSpecifiedTz(data, timeZoneId) {
         if (_.isNil(data)) {
             return null;
         }
 
         if (_.isNumber(data)) {
-            return moment.unix(data).utc().tz(timeZoneId).format('MM/DD/YY'); // TODO/FIXME: Consider using L (although that has a 4 digit year, i.e. YYYY)
+            return moment.unix(data).utc().tz(timeZoneId).format('L LT z');
         }
 
-        return moment(data).tz(timeZoneId).format('MM/DD/YY'); // TODO/FIXME: Consider using L (although that has a 4 digit year, i.e. YYYY)
+        return moment.utc(data, 'YYYY-MM-DDTHH:mm:ss').tz(timeZoneId).format('L LT z');
+    },
+
+    formatDateAndTimeWithTz(data, userTimeZoneId) {
+        const timeZoneId = userTimeZoneId || dateUtils.getDetectedTimeZone();
+
+        return dateUtils.formatDateAndTimeWithSpecifiedTz(data, timeZoneId);
+    },
+
+    formatTime(data) {
+        if (_.isNil(data)) {
+            return null;
+        }
+
+        if (_.isNumber(data)) {
+            return moment.unix(data).utc().format('LT');
+        }
+
+        return moment(data, 'HH:mm').format('hh:mm a');
```

Comment:
I realize the original code was like this, but shouldn't this case also use `.format('LT')`?

Reply:
Yep, but I though maybe there would be to much a regression?

Reply:
I reviewed the documentation (see https://momentjs.com/docs/#/displaying/format/) and  I believe the only difference (at least for the en-US locale) would be whether "AM/PM" is capitalized (as it is with the "LT" standard format) or lower-cased (with the "a" component format).

My own opinion is that we should try and display values in consistent formats, and therefore we should leverage the built-in standard ones, versus custom ones we build from the individual portions of a Date/Time value helps with this.  Also, with internationalization/localization in mind, it is arguably _"wrong"_ to "hard-code" format patterns such as "MM/DD/YY" since certain other locales besides en-US may favor variations such "DD/MM/YY" instead.  The built in standard formats (e.g. "L" for dates) will respect locale settings and do the right thing (see https://momentjs.com/docs/#/i18n/).

Furthermore, in my opinion, there is no logical reason that different forms of the input value (in this case a number representing a UNIX timestamp versus a string representing HH:mm) should result in differently formatted display values.  I cannot think of a valid justification for that behavior, at least off the top of my head.  In all likelihood the original implementation there that does result in this difference was simply an accident.

HOWEVER, all that having been said, your concern is noted, and if your preference for this time is to let it be and not change anything unnecessarily, I fully support that decision.

Reply:
> My own opinion is that we should try and display values in consistent formats, and therefore we should leverage the built-in standard ones

I agree, this is why i renamed them in order to say, "hey, don't use that format like that use our dateUtils format functions."

I am going to go ahead and change it and then merge if you're okay with that.